src/components/ChartContainer/D3Chart.js

```diff
@@ -102,13 +103,16 @@ function D3Chart({
 
     d3.selectAll('.axis-grid line').style('stroke', 'lightgray');
 
-    // X axis label:
-
-    svg.append('text')
-      .attr('x', width / 2)
-      .attr('y', height)
-      .attr('class', 'd3-chart__label')
-      .text('Year to Date');
+    TimelineOptions.map((timeline) => {
```

Comment:
I think a better way to do this would be to use `TimelineOptions.find((option) => option.value === currentTimeline.choice).label`

Reply:
fixed